bio.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://cdn.tailwindcss.com?plugins=forms,typography,aspect-ratio,line-clamp"></script>
+    <script>
+        tailwind.config = {
+            theme: {
+                extend: {
+                    colors: {
+                        clifford: '#da373d',
+                        brown: {
+                            50: '#fdf8f6',
+                            100: '#f2e8e5',
+                            200: '#eaddd7',
+                            300: '#e0cec7',
+                            400: '#d2bab0',
+                            500: '#bfa094',
+                            600: '#a18072',
+                            700: '#977669',
+                            800: '#846358',
+                            900: '#43302b',
+                        },
+                        blue: {
+                            850: '#17275c',
+                        },
+                    }
+                }
+            }
+        }
+    </script>
+    <style type="text/tailwindcss">
+        @layer utilities {
+            .content-auto {
+                content-visibility: auto;
+            }
+        }
+        /* .btn {
+            @apply px-5 py-2 inline-block bg-blue-850 text-white font-medium rounded shadow hover:bg-brown-700 hover:text-black active:bg-clifford active:text-white;
+        } */
+    </style>
+</head>
+
+<body>
+    <header>
+        <nav class="bg-gray-800">
```

Comment:
Consider adding a personal touch to your nav bar, or adding a footer at the bottom of each page. That way, it is clear that this is distinctly your site, no matter what page you're on.

Reply:
I totally agree. I wish I could have spend more time.